js/client-battle-tooltips.js

```diff
@@ -1206,6 +1271,10 @@ var BattleTooltips = (function () {
 		// Type-enhancing items
 		if (itemTypes[item.name] === moveType) return this.battle.gen < 4 ? 1.1 : 1.2;
 
+		// Pokemon-specific items
+		if (item.name === 'Soul Dew' && this.battle.gen < 7) return 0;
+		if (orbTypes[item.name] === moveType || moveType === 'Dragon') return 1.2;
```

Comment:
If I'm reading this correctly, this will make all Dragon-type moves 1.2x damage?